assembl/static2/package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "dev": "webpack -d --watch",
-    "build": "webpack -p",
+    "dev": "webpack -d --watch --progress --colors",
```

Comment:
No we're on the same version. Strange, I didn't see these options in the `--help`. But I did see it in the documentation online. I doubt that it affects the build logs at all. So all good 😄